src/test/java/seedu/placebook/logic/commands/EditAppCommandTest.java

```diff
@@ -0,0 +1,170 @@
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.placebook.logic.commands.CommandTestUtil.DESC_A;
+import static seedu.placebook.logic.commands.CommandTestUtil.DESC_B;
+import static seedu.placebook.logic.commands.CommandTestUtil.VALID_DESCRIPTION_A;
+import static seedu.placebook.logic.commands.CommandTestUtil.VALID_LOCATION_B;
+import static seedu.placebook.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.placebook.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.placebook.logic.commands.CommandTestUtil.showAppointmentAtIndex;
+import static seedu.placebook.testutil.TypicalIndexes.INDEX_FIRST_APPOINTMENT;
+import static seedu.placebook.testutil.TypicalIndexes.INDEX_SECOND_APPOINTMENT;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.placebook.commons.core.Messages;
+import seedu.placebook.commons.core.index.Index;
+import seedu.placebook.logic.UiStubFactory;
+import seedu.placebook.model.Contacts;
+import seedu.placebook.model.Model;
+import seedu.placebook.model.ModelManager;
+import seedu.placebook.model.UserPrefs;
+import seedu.placebook.model.person.Person;
+import seedu.placebook.model.schedule.Appointment;
+import seedu.placebook.model.schedule.Schedule;
+import seedu.placebook.testutil.AppointmentBuilder;
+import seedu.placebook.testutil.ContactsBuilder;
+import seedu.placebook.testutil.EditAppDescriptorBuilder;
+import seedu.placebook.testutil.PersonBuilder;
+import seedu.placebook.testutil.Seed;
+import seedu.placebook.ui.Ui;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for EditAppCommand.
+ */
+public class EditAppCommandTest {
+    // default positive confirmation ui. This will not affect EditAppCommand
+    private final Ui uiStub = UiStubFactory.getUiStub(true);
+
+    private Model model;
+    private Contacts testContacts;
+    private Schedule testSchedule;
+
+    @BeforeEach
+    public void setUp() {
+        Person client1 = new PersonBuilder().withName("Client1").build();
+        Person client2 = new PersonBuilder().withName("Client2").build();
+
+        testContacts = new ContactsBuilder().build();
+        testContacts.addPerson(client1);
+        testContacts.addPerson(client2);
+
+        Appointment appointment1 = new AppointmentBuilder(Seed.ONE).addClient(client1).build();
+        Appointment appointment2 = new AppointmentBuilder(Seed.THREE).addClient(client1).addClient(client2).build();
+
+        testSchedule = new Schedule();
+        testSchedule.addAppointment(appointment1);
+        testSchedule.addAppointment(appointment2);
+        model = new ModelManager(testContacts, new UserPrefs(), testSchedule);
+        System.out.println(model.getFilteredAppointmentList().get(0));
+        System.out.println(model.getFilteredAppointmentList().get(1));
+    }
+
+    @Test
+    public void execute_someFieldsSpecifiedUnfilteredList_success() {
```

Comment:
should there be a filteredList success testcase?

Reply:
I'll add it in, thanks!